src/r_util.cpp

```diff
@@ -434,6 +432,15 @@ namespace rhost {
             return R_NilValue;
         }
 
+        extern "C" SEXP toJSON(SEXP obj) {
+            SEXP json = Rf_mkCharCE(to_json(obj).serialize().c_str(), CE_UTF8);
```

Comment:
Should you use rhost::util::exceptions_to_errors to protect against exceptions from `to_json`, as it uses picojson and std which may throw?